tests/selenium/options_test.py

```diff
@@ -28,6 +34,32 @@ def add_test_origin(self, origin, action):
         self.js("badger.storage.setupHeuristicAction('{}', '{}');".format(
             origin, action))
 
+    def add_test_origins(self, origins_with_actions):
+        """Add a dictionary of origins with their actions to backend storage."""
+        self.load_options_page()
+        for origin in origins_with_actions:
+            self.js("badger.storage.setupHeuristicAction('{}', '{}');".format(
+                origin, origins_with_actions[origin]))
+
+    def user_overwrite(self, origin, action):
+        # Get the slider that corresponds to this radio button
+        origin_div = self.driver.find_element_by_css_selector('div[data-origin="' + origin + '"]')
+        slider = origin_div.find_element_by_css_selector('.ui-slider')
+
+        # Click on the correct place over the slider to block this origin
+        click_action = ActionChains(self.driver)
+        if action == 'block':
+            # Top left (+2px)
+            click_action.move_to_element_with_offset(slider, 2, 2)
```

Comment:
Do we need to depend on slider dimensions in order to click them? This seems fragile.

Reply:
Ah, I see, but you should be able to do it with JavaScript, which seems more clear/maintainable. Something like:
```python
self.js("$('#block-{}').click()".format(DOMAIN.replace(".", "-")))
```

Reply:
For example: https://github.com/EFForg/privacybadger/blob/94588e71a63964c0274f6fb7827c369b5c56a8a4/tests/selenium/popup_test.py#L226-L228

Reply:
Ah, I see, we can't do this yet on the options page because of how saving is triggered at the moment there (I think).